src/context.js

```diff
@@ -61,6 +61,24 @@ angular.module('hypermedia')
       }},
 
       /**
+       * Refresh all resources in the context.
+       *
+       * @function
+       * @returns {Promise} a single promise that will be resolved with an array/hash of values.
```

Comment:
Met het gevaar te miepen: het is niet erg duidelijk waar de promise nu mee geresolved wordt. Wat dacht je van: "Returns a promise that will be resolved with an array of all resources that were refreshed. If any of the requests fail, the promise will be rejected with the response."

In plaats van de beschrijving misschien: "Refresh all resources in the context by issuing a HTTP GET request on them."